### PR TITLE
gru kernel config fix

### PR DESCRIPTION
**Pull request by WizzardSK**
with multiarch and snap support

### CODE REVIEW

**Comment by Maccraft123** [comment]
This change is not needed as it was added in one of previous commits

**Comment by WizzardSK** [comment]
ok, but this change at least unbreaks the gru config, so Cadmium is bootable on kevin with this change

**Comment by Maccraft123** [comment]
Then fix it properly, adding a driver for mmc to config is 5 line change, not 1k line change

**Comment by WizzardSK** [comment]
I do not understand what to fix. All I did was reverted that problematic commit for gru config that made kevin unbootable and added CONFIG_COMPAT and SQUASHFS support.

**Comment by drocsid** [comment]
@WizzardSK looking at the code review it looks like you are rolling back kernel changes to kernel rc1. This probably rolled back some changes @Maccraft123 made when she pushed rc3. Then I'd suggest starting from rc3.

**Comment by WizzardSK** [comment]
@drocsid yes, that is exactly what I told. I had to revert that, because kevin is not booting after the latest change, like I said before. Plus I added CONFIG_COMPAT and SQUASHFS support.

File: baseboard/gru/kernel-config [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/arm64 5.14.0-rc3 Kernel Configuration
+# Linux/arm64 5.14.0-rc1 Kernel Configuration
```
Comment (drocsid) [inline baseboard/gru/kernel-config:3]: going back from rc3 => rc1. But current master is rc3 

https://github.com/Maccraft123/Cadmium/blob/master/baseboard/gru/kernel-config

**Comment by drocsid** [comment]
> @drocsid yes, that is exactly what I told. I had to revert that, because kevin is not booting after the latest change, like I said before. Plus I added CONFIG_COMPAT and SQUASHFS support.

I think I understand where you are coming from. If you want to protect users of kevin from the not booting issue perhaps create an issue and reference this branch. Then they might fix it there. You could also track when the issue is resolved from there.

**Pull request closed without merging**